cli/atlas/templates/cmd/config.go.gotmpl

```diff
@@ -8,5 +8,7 @@ const (
   GatewayAddress = "0.0.0.0:8080"
   // GatewayURL is the default URL endpoint for the gateway server.
   GatewayURL = "/{{ .Name | URL }}/v1/"
-  {{ end }}
+  {{ end }}{{ if .WithDatabase -}}
+  // DatabaseAddress is the default address for the database, if no override is specified in the flags
+  DatabaseAddress = "0.0.0.0:5432"{{ end }}
```

Comment:
this should be `DatabaseConnectionString` or `DBConnectionString` instead of address, and default to a postgres connection string to a database named after the generated `-name` for the project:
`"host=localhost port=5432 dbname=<app name>"`

There might be some TLS thing you'll need in there, too...

And this should probably also include running a postgres docker image with the same configuration in the Makefile

Reply:
*The name thing:* Good point -- it could be weird. Maybe spend a few seconds to see what kind of database naming restrictions there are for postgres? I think we should keep to just the `-name` parameter and let it look weird rather than adding a new name -- that can be a problem for Future Zach to worry about :)

*The TLS thing:* I meant there should be a `sslmode=disable` setting in the connection string, like mentioned [here](https://github.com/xo/usql/issues/6). I don't know if it's necessary, but it was for me out-of-the-box. It could have been that I did something dumb, though :)

*postgres docker thing:* yep, ignore this in favor of getting a bootstrapped manifest. Maybe that can be worked on really soon to obviate this need.